src/sscofpmf.adoc

```diff
@@ -183,7 +183,7 @@ M-mode).
 
 Read access to bit _X_ is subject to the same mcounteren (or mcounteren and
 hcounteren) CSRs that mediate access to the hpmcounter CSRs by S-mode (or
-VS-mode). In M and S modes, scountovf bit _X_ is readable when mcounteren bit
+VS-mode). In S modes, scountovf bit _X_ is readable when mcounteren bit
```

Comment:
@gfavor Don't we need to say what happens in M-mode, e.g. add a sentence before this one like "In M-mode, scountovf bit _X_ is always readable."

Reply:
Yes re scountovf.  @wmat , the overall change should be replacing the "In M and S modes, ..." sentence with the following two sentences:

_In M mode, scountovf bit X is always readable.  In S/HS modes, scountovf bit X is readable when mcounteren bit X is set, and otherwise reads as zero._

Reply:
I've made the change as described, however, is S/HS correct?  Should it be S/VS?
```suggestion
VS-mode). In M mode, scountovf bit X is always readable, in S/HS modes, scountovf bit _X_ is readable when mcounteren bit
```